### PR TITLE
Add 20+ sources and organizations by Tapclicks

### PR DESCRIPTION
**Pull request by djbange**

### CODE REVIEW

File: organizations/vertical_response.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "VERTICAL_RESPONSE",
+  "name": "verticalresponse",
```
Comment (pfrisella) [inline organizations/vertical_response.json:3]: VerticalResponse

File: organizations/yasabe.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "YaSabe",
+  "id": "YaSabe",
```
Comment (pfrisella) [inline organizations/yasabe.json:3]: capitalize the ID

File: sources/strata.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "STRATA",
+  "name": "Strata",
+  "categories": ["PAYMENTS", "FINANCE", "ANALYTICS"],
+  "organization": "FREEWHEEl",
```
Comment (pfrisella) [inline sources/strata.json:5]: FREEWHEEL

File: sources/wufoo.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "wufoo",
```
Comment (pfrisella) [inline sources/wufoo.json:2]: WUFOO

File: sources/ya_sabe.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "YA_SABE",
+  "name": "YaSabe",
+  "categories": ["MARKETING", "SEO", "JOBS", "ADVERTISING"],
+  "organization": "YaSabe",
```
Comment (pfrisella) [inline sources/ya_sabe.json:5]: YASABE

File: organizations/freewheel.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "FREEWHEEl",
```
[the remark this answers is not in the document]
Comment (djbange) [reply organizations/freewheel.json]: Thanks for the feedback. I updated the requested changes.

**Pull request merged**